src/Compilers/CSharp/Test/Syntax/Parsing/ParserErrorMessageTests.cs

```diff
@@ -4353,6 +4353,28 @@ static int Main(string[] args)
     Diagnostic(ErrorCode.ERR_RbraceExpected, ":"));
         }
 
+        [Fact]
+        public void CS1525ERR_InvalidExprTerm_Location()
+        {
+            var test = @"
+public class Repro_Cs1525
+{
+    public void Method()
+    {
+        var v = 42;
+        >>;
+    }
+}
+";
+            ParseAndValidate(test,
+                    // (7,9): error CS1525: Invalid expression term '>'
+                    //         var v = 42;
```

Comment:
>//         var v = 42; [](http://example.com/codeflow?start=20&length=22)

It looks like the comment is out of sync. Please update all comments to actually reflect the current behavior.